src/components/pullRequests/card/card.module.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  border: 0.2em solid black;
+  border-radius: 0.5em;
+  padding: 1em;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  max-width: 850px;
+  margin-bottom: 2em;
+}
+
+.prTitle {
+  color: #041484;
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+.cardFooter {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.profilePicture {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 6em;
+}
+
+.profilePicture > img {
+  width: 3em;
+  border-radius: 50%;
+}
+
+.statusLable {
+  color: grey;
+  font-weight: bold;
+}
+
+.activeBtn{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #4fb90d;
+  color: white;
+  border: 2px solid black;
+  border-radius: 5px;
+  padding: 0.5rem 2rem;
```

Comment:
Please use px as a unit for border, margin, padding, these metrics are not dependent on the parents/roots font-size.
https://css-tricks.com/rems-ems/